ios-continuous-integration_5c8a397c2c7d3a154460db00.md

```diff
@@ -19,63 +22,58 @@ types][machine-types].
   - [Fastfile](#fastfile)
   - [Building your app](#building-your-app)
   - [Releasing your app](#releasing-your-app)
-- [Run the demo project yourself](#run-the-demo-project-yourself)
-
-## Demo project
+- [Run the demo project yourself](#run-the-example-project-yourself)
 
-Semaphore maintains an example iOS Swift project:
+## Example project
 
-- [Demo Swift Xcode project on GitHub][demo-project]
+Semaphore maintains an example iOS app written in Swift 5.1 and SwiftUI to
+demonstrate how to get an Xcode CI/CD environment up and running:
 
-In the repository you will find an annotated Semaphore configuration file
-`.semaphore/semaphore.yml`.
+- [TallestTowers example project on GitHub][example-project]
 
-The application uses Swift, Xcode and Fastlane with the [Semaphore
-plugin][fastlane-plugin].
+This example project includes an annotated Semaphore configuration file,
+[`.semaphore/semaphore.yml`](https://github.com/semaphoreci-demos/semaphore-demo-ios-swift-xcode/blob/master/.semaphore/semaphore.yml) and uses [fastlane](fastlane) for its CI pipeline with the
```

Comment:
point to new repo https://github.com/semaphoreci-demos/semaphore-demo-ios-swift-xcode-v2 after importing code